packages/web-workers/src/messenger/index.ts

```diff
@@ -1,2 +1,6 @@
 export {createWorkerMessenger} from './worker';
-export {createIframeWorkerMessenger} from './iframe';
+export {
+  createNullOriginIframeMessenger as createIframeWorkerMessenger,
+  createNullOriginIframeMessenger,
+  createRemoteOriginIframeMessenger,
+} from './i-frame';
```

Comment:
`i-frame` is kind of strange...

Reply:
[I agree](https://github.com/Shopify/remote-ui/pull/25#issuecomment-702916291). It's self defense for now 😬

Reply:
I think you can purge the cache by adding a version after the `-typescript-` bit of these lines: https://github.com/Shopify/remote-ui/blob/main/.github/workflows/ci.yml#L39-L41

```
key: ${{ runner.os }}-typescript-v2-${{ github.sha }}
restore-keys: |
  ${{ runner.os }}-typescript-v2-
```

Reply:
pushed some updates according to your comments.

I can't seem to change the ci.yml file, though.

```
 ! [remote rejected] iframe-messenger -> iframe-messenger (refusing to allow an OAuth App to create or update workflow `.github/workflows/ci.yml` without `workflow` scope)
```

Got to figure out how to fix that first.

Reply:
@lemonmade not sure if that's a privilege I may be missing for the repo?